extension-ui/src/components/workspace/estimateCost/index.js

```diff
@@ -0,0 +1,69 @@
+import {
+  StructuredListWrapper,
+  StructuredListHead,
+  StructuredListBody,
+  StructuredListRow,
+  StructuredListCell,
+  Tooltip,
+} from 'carbon-components-react';
+const mockData = require('./mock.json');
+
+
+const EstimateCost = ({ result }) => {
+    result = result ? result : mockData;
+    
+    return (
+        <div className="bx--cost_detail">
+            <h2>Cost Estimate</h2>
+            <div>
+                <StructuredListWrapper>
+                    <StructuredListHead>
+                        <StructuredListRow head>
+                            <StructuredListCell head>Resource</StructuredListCell>
+                            <StructuredListCell head>Local Name</StructuredListCell>
+                            <StructuredListCell head>Title</StructuredListCell>
+                            <StructuredListCell head>Current Cost</StructuredListCell>
+                            <StructuredListCell head>Previous Cost</StructuredListCell>
+                            <StructuredListCell head>Changed Cost</StructuredListCell>
+                        </StructuredListRow>
+                    </StructuredListHead>
+                
+                    <StructuredListBody>
+                        {result?.Lineitem.map((item, index) => (
+                        <StructuredListRow key={index}>
+                            <StructuredListCell>{item.terraformItemId}</StructuredListCell>
+                            <StructuredListCell>{item.id}</StructuredListCell>
+                            <StructuredListCell>{item.title}</StructuredListCell>
+                            <StructuredListCell>{item.currlineitemtotal.toFixed(2)} {result?.currency}{item.rateCardCost?"*":""}</StructuredListCell>
+                            <StructuredListCell>{item.prevlineitemtotal.toFixed(2)} {result?.currency}{item.rateCardCost?"*":""}</StructuredListCell>
+                            <StructuredListCell>{item.changelineitemtotal.toFixed(2)} {result?.currency}{item.rateCardCost?"*":""}</StructuredListCell>
+                        </StructuredListRow>
+                        ))}
+                        <StructuredListRow key={result?.Lineitem.length+1}></StructuredListRow>
+                        <StructuredListCell head>  
+                            <Tooltip
+                                direction="bottom"
+                                triggerText="Total estimated cost"
+                            >
+                                <p style={{padding:'2px',fontSize:"15px" ,width:'170px',background: '#0f62fe'}}>
+                                    The cost displayed here are just an estimated cost not an actual cost.
+                                    * indicated cost has been derived from ratecard
+                                </p>
+                            </Tooltip>
```

Comment:
![image](https://user-images.githubusercontent.com/1298628/151342752-a0c1d9c5-e2f1-4ae5-b426-dd02e796dc47.png)

- The tooltip is not showing as tooltip and displaying with blue background
- The tooltip info icon needs to be beside the **Total estimated cost** and not below  
- If we cannot make the tooltip work, just have a helper text below the **Total estimated cost**

Reply:
Tooltip was causing some design issues so we removed it and added a helper text in footer of the webview